pkg/catalogv2/helm/helm3.go

```diff
@@ -15,10 +19,13 @@ import (
 	"helm.sh/helm/v3/pkg/release"
 )
 
+// isHelm3 checks if the value of the owner key of the received map is equal to helm
```

Comment:
// isHelm3 checks if the value of the owner key of the received map is equal to helm
// since every helm release object has this particular label on it.